systest/build.rs

```diff
@@ -15,7 +15,11 @@ fn main() {
                 s.to_string()
             }
         })
-        .skip_type(|t| t.ends_with("FUNC"))
-        .skip_fn(|f| f == "libssh2_userauth_password_ex" || f == "libssh2_session_init_ex");
+        .skip_type(|t| t.ends_with("FUNC") || t.contains("KBDINT"))
+        .skip_fn(|f| {
+            f == "libssh2_userauth_password_ex"
+                || f == "libssh2_session_init_ex"
+                || f == "libssh2_userauth_keyboard_interactive_ex"
```

Comment:
@alexcrichton systest seems to not like the callback types, so I added this to the skip list here

Reply:
Ok ,that seems fine so long as it's been manually verified